src/pages/LoadUnload/contexts/mapLayersReducer.ts

```diff
@@ -1,21 +1,28 @@
-import { loadUnloadSpaces } from 'shared/map/mapLayers'
-
-export const roadSectionsLoadUnloadLayerId = 'roadSectionsLoadUnload'
+import { MapLayerId } from './MapContext'
 
 export const mapLayersInitialState = {
-  [loadUnloadSpaces.id]: true,
-  [roadSectionsLoadUnloadLayerId]: true,
+  [MapLayerId.loadUnloadLayerId]: true,
+  [MapLayerId.roadSectionsLoadUnloadLayerId]: true,
+  [MapLayerId.bollardsLayerId]: false,
 }
 
-export const layerIds = ['loadUnloadSpaces', 'roadSectionsLoadUnload'] as const
-
 export type mapLayerActionType = {
-  type: 'TOGGLE'
-  layerId: (typeof layerIds)[number]
+  type: 'ON' | 'OFF' | 'TOGGLE'
```

Comment:
Is type the most clear name for this property?

Reply:
This reducer function is existing functionality, and I'm trying not to change too much about it.